helper/general_objects/schemas.go

```diff
@@ -310,7 +310,10 @@ func DefinitionAttributeSchema(excludeTypes []string, excludeFields []string, fo
 
 		// Enum only
 		"options": {
-			Type:        schema.TypeMap,
+			Type: schema.TypeMap,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+			},
```

Comment:
Not related to this ticket, but all enum options parameters should be a map of String.